scripts/benchmarks/parse-benchmarks-data.js

```diff
@@ -15,7 +15,8 @@ var BENCHMARK_CSV_COLUMNS = [
   'decile8',
   'decile9',
   'decile10',
-  'isToppedOut'
+  'isToppedOut',
+  ''
```

Comment:
is this so you have a trailing comma in the generated csv? might be worth having a comment here

Reply:
Ya it is, I'll add a comment.